### PR TITLE
feat(service-portal): Implement standard error/empty states in ui

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

Standardized error/empty state for (almost) every screen in the service portal
Licenses and occupational licenses will be updated later on since they're a bit different

Also updating the `husnaedis-og-mannvirkjastofnun` -> `hms` slug since it's out of sync with the contentful slug

## Why

So each module in the service portal doesn't reinvent the error state wheel every time.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/education/src/screens/SecondarySchoolGraduationDetail/SecondarySchoolGraduationDetail.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import { useGetInnaDiplomasQuery } from '../SecondarySchoolCareer/Diplomas.gener
 import { tagSelector } from '../../utils/tagSelector'
 import { defineMessage } from 'react-intl'
```
Comment (thordurhhh) [inline libs/service-portal/education/src/screens/SecondarySchoolGraduationDetail/SecondarySchoolGraduationDetail.tsx:23]: unused imports here

**Review by thordurhhh — APPROVED** [review]
Nice one ⛷️

File: libs/react-spa/shared/src/components/problems/Problem.tsx [hunk 2]
```diff
@@ -114,7 +114,7 @@ export const Problem = ({
           return (
             <ThirdPartyServiceError
               organizationSlug={organizationSlug}
-              size={size}
+              size={size ?? 'small'}
```
Comment (saevarma) [inline libs/react-spa/shared/src/components/problems/Problem.tsx:117]: This is changing the default sizing from `large` to `small` for everyone using the shared component. I´m not sure that is what we want. What is the goal here? When do you need it small?

Comment (thorkellmani) [reply libs/react-spa/shared/src/components/problems/Problem.tsx:117]: According to the designs when the error is from a specific service provider, we want to display a small warning. 
image
However, when the error doesn't come from a specific source, we want the large box.
image

Somehow overlooked how far-reaching this change would be.

**Review by saevarma — APPROVED** [review]
The Core file LGTM

**Pull request merged**